### ISSUE TITLE
Are these cookies signed ?

### ISSUE THREAD

**Issue by javajack** [LCTL/java-stateless-http-session#1]
If I am not wrong we need to implement the signed cookies for the sake of implementing security in a right manner.

Does you code implement signed cookies ? Can you add an example project as well ?

**Comment by LCTL** [comment]
YES. It is signed cookies. I will provide more example in README
